README.md

```diff
@@ -42,6 +42,19 @@ For async communication and long running discussions please use issues and pull
 
 For sync communication we have an IRC channel #cri-o, on chat.freenode.net, that everyone is welcome to join and chat about development.
 
+## Compatibility matrix: CRI-O <-> Kubernetes clusters
+
+| Version - Branch        | Kubernetes branch/version     | Maintenance status |
+|-------------------------|-------------------------------|--------------------|
+| CRI-O 1.0  - kube-1.6.x | Kubernetes 1.6 branch, v1.6.x | =                  |
```

Comment:
So this looks like as if 1.0 is already released and the `kube-1.6.x branch` is already in maintenance mode.